components/list/test/list.test.js

```diff
@@ -147,12 +147,30 @@ describe('d2l-list', () => {
 		beforeEach(async() => {
 			elem = await fixture(html`
 				<d2l-list>
-					<d2l-list-item selectable key="L1-1" label="L1-1"></d2l-list-item>
+					<d2l-list-item selectable key="L1-1" label="L1-1">
+						<div>Content</div>
+						<div><d2l-button>Button</d2l-button></div>
+					</d2l-list-item>
 					<d2l-list-item selectable key="L1-2" label="L1-2"></d2l-list-item>
 				</d2l-list>
 			`);
 		});
 
+		it('does not dispatch d2l-list-selection-changes event when interactive element is clicked in selectable item', async() => {
+			let dispatched = false;
+			elem.addEventListener('d2l-list-selection-changes', () => dispatched = true);
+			const button = elem.querySelector('d2l-button');
+			setTimeout(() => clickElem(button));
+			await oneEvent(button, 'click');
+			expect(dispatched).to.equal(false);
+		});
+
+		it('dispatches d2l-list-selection-changes event when non-interactive element is clicked', async() => {
```

Comment:
Confirmed that this test would have caught [this](https://desire2learn.atlassian.net/browse/GAUD-7865) reported defect (fails on webkit on main)